lib/motors/controller.hpp

```diff
@@ -1,19 +1,40 @@
 #pragma once
+
 #include <cstdint>
+#include <optional>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include <rapidjson/document.h>
+#include <rapidjson/error/en.h>
+#include <rapidjson/istreamwrapper.h>
 
 #include "core/logger.hpp"
+#include "core/types.hpp"
 
 namespace hyped::motors {
 enum class ControllerStatus { kControllerTemperatureExceeded, kUnrecoverableWarning, kNominal };
+// every frame sent to the controller contains 8 bytes
+static constexpr std::uint8_t kControllerCanFrameLength = 8;
```

Comment:
is this a more global constant? will all our can frames be of this length? if so, move this to `io/can.hpp`